src/common/options/rgw.yaml.in

```diff
@@ -4294,6 +4294,39 @@ options:
   services:
   - rgw
   with_legacy: true
+- name: rgw_http_notif_message_timeout
```

Comment:
can we document these options somewhere? either in a 'config' section in https://docs.ceph.com/en/latest/radosgw/notifications/ or a new 'bucket notification' section similar to https://docs.ceph.com/en/latest/radosgw/config-ref/#topic-persistency-settings

and note that when rendered that way, it only shows the `long_desc`